tensorflow/compiler/xla/stream_executor/stream.cc

```diff
@@ -1590,6 +1703,12 @@ Stream &Stream::ThenBlasGemmBatched(
     uint64_t k, float alpha, DeviceMemorySlice<Eigen::half> a, int lda,
     DeviceMemorySlice<Eigen::half> b, int ldb, float beta,
     DeviceMemorySlice<Eigen::half> c, int ldc, int batch_count, blas::CallContext context) {
+  if (gpu::GpuBlasLtEnabled()) {
```

Comment:
I think we should put the call to BlasLtRunner to **ThenBlasGemmBatchedWithScratch** functions instead. Because ThenBlasGemmBatched functions are forwarding calls.

Then, in ThenBlasGemmBatchedWithScratch we can provide a (non-null) allocator pointer to RunBatched calls

Reply:
@pemeliya Thanks for the comment. I have updated the code : )